### PR TITLE
September key features

### PR DESCRIPTION
**Pull request by NolwennP**
Add 2 new Serenity key features delivered in September in the PIM.

### CODE REVIEW

File: content/whats-new/Serenity-news.json [hunk 1]
```diff
@@ -132,6 +139,13 @@
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-07.html#easily-fix-errors-on-your-connections"
   },
+  {
+    "title":"Rule duplication 🤖",
+    "domains": ["Automation","Productivity"],
+    "editions": ["CE","EE"],
```
Comment (LaureBro) [inline content/whats-new/Serenity-news.json:145]:
```suggestion
    "editions": ["EE"],
```

File: content/whats-new/Serenity-news.json [hunk 2]
```diff
@@ -13,6 +13,13 @@
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-07.html#new-create-and-update-rules-from-the-pim"
   },
+  {
+    "title":"Attribute and option labels in product XLSX and CSV exports",
```
Comment (LaureBro) [inline content/whats-new/Serenity-news.json:17]:
```suggestion
    "title":"Attributes and options labels displayed in XLSX and CSV product exports",
```

Comment (LaureBro) [reply content/whats-new/Serenity-news.json:17]: or "New option to display labels in the XLSX and CSV product exports"

Comment (NolwennP) [reply content/whats-new/Serenity-news.json:17]:
```suggestion
    "title":"Labels displayed in XLSX and CSV product exports",
```

**Review by LaureBro — APPROVED** [review]

**Pull request merged**